countdown/module.py

```diff
@@ -0,0 +1,256 @@
+from datetime import datetime
+from typing import Optional, List
+
+import dateutil.parser
+import dateutil.relativedelta
+import discord
+from discord.ext import commands
+
+from pie.utils import ConfirmView
+from .database import CountdownItem
+from pie import check, i18n, logger, utils
+
+_ = i18n.Translator("modules/fun").translate
+bot_log = logger.Bot.logger()
+guild_log = logger.Guild.logger()
+
+
+class Countdown(commands.Cog):
+    """Countdown"""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    async def _process_text(self, ctx: commands.Context, text: Optional[str]):
```

Comment:
This should be a utils function in pumpkin-py core. I think I've seen this kind of code in other modules / pumpkin-py repos.